src/alt-safe.ts

```diff
@@ -0,0 +1,94 @@
+import SafeApiKit, { SafeInfoResponse } from "@safe-global/api-kit";
```

Comment:
Would love to have avoided this too... its a checky hack to determine if the safe the user wants to use exists already or not. This could be determined by the setup (i.e. seed) data.